File: vbSample/Form1.vb
Public Class Form1
   Public Sub New()

      ' This call is required by the designer.
      InitializeComponent()

      AxPDFiumControl2.Document = "http://google.com"

   End Sub

   Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
      openFileDialog1.ShowDialog()
      '      AxPDFiumControl2.Document = openFileDialog1.FileName
      AxPDFiumControl2.Document = "http://google.com"

   End Sub

   Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
      AxPDFiumControl2.Document = "http://google.com"

   End Sub
End Class

File: cSharpSample/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace cSharpSample {
   public partial class Form1:Form {
      public Form1() {
      InitializeComponent();
      }

      private void button1_Click(object sender,EventArgs e) {

          DialogResult rc = openFileDialog1.ShowDialog();

          if ( DialogResult.OK == rc )
            axPDFiumControl1.Document = openFileDialog1.FileName;
      }
   }
}
